tests/snmpsim/netscaler-sdx.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.5951.6
```

Comment:
I've searched for that strange hostname and found it in some support cases. I think it is just the host where the SDX software has been compiled on.